integration_test/cases/migrator.exs

```diff
@@ -81,6 +86,9 @@ defmodule Ecto.Integration.MigratorTest do
 
       assert count_entries() == 1
       assert [50] = run(TestRepo, path, :up, to: 50, log: false)
+
+      purge migrations
```

Comment:
If you remove the `purge` but keep the `run(TestRepo, path, :down, all: true, log: false)` that I added to clean up after each test you'll get warnings in both postgres and mysql.